### PR TITLE
Feedback from Peter

### PR DESCRIPTION
**Pull request by iaindillingham**
Three improvements to the test suite, following feedback from @inglesp.

### CODE REVIEW

**Review by LFISHER7 — APPROVED** [review]
Looks good to me!

File: tests/test_cohort_joiner.py [hunk 1]
```diff
@@ -41,7 +42,16 @@ class TestWriteDataframe:
     def test_write_csv(self, tmp_path, dataframe):
         csv_path = tmp_path / "input.csv"
         cohort_joiner.write_dataframe(dataframe, csv_path)
-        assert csv_path.exists()
+        # When writing a dataframe to a CSV, it's easy to write the dataframe's index,
```
Comment (milanwiedemann) [inline tests/test_cohort_joiner.py:45]: Oh thanks, this is nice! @ccunningham101 and I ran into this problem while thinking about the decile chart action

**Comment by iaindillingham** [comment]
If you're happy, would you mind giving the ✅ either @LFISHER7 or @milanwiedemann? Thanks 🙂

**Review by milanwiedemann — APPROVED** [review]

**Comment by milanwiedemann** [comment]
sorry forgot to click approve

**Pull request merged**